### PR TITLE
footer added

### PR DESCRIPTION
**Pull request by pankajbijarniya**
Solved issue: #5 

Footer added with social media icons.
You can replace dummy data with your original data and social media links.

This footer is completely responsive.

screencapture-3000-silver-fish-2yydtxe7-ws-us03-gitpod-io-2021-02-14-12_13_34

### CODE REVIEW

**Comment by pankajkumarbij** [comment]
> Solved Issue: #5  
Footer added with social media icons.
> You can replace dummy data with your original data and social media links.
> 
> This footer is completely responsive.
> 
> screencapture-3000-silver-fish-2yydtxe7-ws-us03-gitpod-io-2021-02-14-12_13_34

File: server/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,8634 @@
 {
   "name": "server",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (fave77) [inline server/package-lock.json:4]: Changes in `server/package-lock.json` aren't required for issue #5

File: client/src/App.js [hunk 2]
```diff
@@ -63,7 +65,10 @@ class App extends Component {
 
           </Switch>
         </div>
+        <br/><br/><br/>
```
Comment (fave77) [inline client/src/App.js]: Render the footer only in the `Home` component

File: client/package.json [hunk 3]
```diff
@@ -8,6 +8,10 @@
     "eject": "react-scripts eject"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
```
Comment (fave77) [inline client/package.json:11]: Do not include social media icons

**Review by fave77 — CHANGES_REQUESTED** [review]
@pankajkumarbij Please adhere to the basic requirements mentioned in issue #5 
If you want to make some additional inputs, first discuss it on discord.

**Review by fave77 — CHANGES_REQUESTED** [review]
@pankajkumarbij
- This PR shouldn't change `package.json` or `package-lock.json`
- You don't need to use any external library
- Just add the required 3 items since the goal is to keep the design minimal
- Please discuss the design decisions over Discord before making PR

**Comment by pankajbijarniya** [comment]
I will make a new PR again.

**Pull request closed without merging**